static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx

```diff
@@ -102,39 +137,75 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
               disableLink
             />
           }
-          triggerProps={{'aria-label': currentProject.slug}}
+          onChange={option => setCurrentProject(option.value)}
+          triggerProps={{'aria-label': currentProject?.slug}}
+          options={projectSelectOptions}
           position="bottom-end"
         />
-        <OnboardingContent currentProject={currentProject} />
+        {currentProject && (
+          <OnboardingContent
+            currentProject={currentProject}
+            isSupported={supportedProjects.includes(currentProject)}
+          />
+        )}
       </TaskSidebarList>
     </TaskSidebar>
   );
 }
 
-function OnboardingContent({currentProject}: {currentProject: Project}) {
-  const currentPlatform = platforms.find(p => p.id === currentProject.platform);
+function OnboardingContent({
+  currentProject,
+  isSupported,
+}: {
+  currentProject: Project;
+  isSupported: boolean;
+}) {
+  const currentPlatform = platforms.find(p => p.id === currentProject?.platform);
   const api = useApi();
   const organization = useOrganization();
   const [received, setReceived] = useState(false);
   const previousProject = usePrevious(currentProject);
   useEffect(() => {
+    if (!currentProject || !previousProject) {
+      return;
+    }
     if (previousProject.id !== currentProject.id) {
       setReceived(false);
     }
-  }, [previousProject.id, currentProject.id]);
+  }, [currentProject, previousProject]);
 
   const docKeysMap = useMemo(() => makeDocKeyMap(currentPlatform?.id), [currentPlatform]);
 
   const {docContents, isLoading, hasOnboardingContents} = useOnboardingDocs({
     docKeys: docKeysMap ? Object.values(docKeysMap) : [],
     project: currentProject,
-    isPlatformSupported: true,
+    isPlatformSupported: isSupported,
   });
 
   if (isLoading) {
     return <LoadingIndicator />;
   }
 
+  if (!isSupported) {
+    // this content will only be presented if the org only has one project and its not supported
+    // in these scenarios we will auto-select the unsupported project and render this message
+    return (
+      <Fragment>
+        <div>
+          {tct(
+            'Fiddlesticks. Profiling isn’t available for your [platform] project yet but we’re definitely still working on it. Stay tuned.',
```

Comment:
Can we say something like "reach out to profiling@sentry.io" or "ask on discord about platform support". I assume there may be a good part of platforms we wont support and dont want to give users the false impression that we are working on it

Reply:
Yeah good point, i'll fix it up